README.md

```diff
@@ -204,13 +204,28 @@ class Node extends libp2p {
 
 ### API
 
-#### Create a Node - `new libp2p.Node(options)`
+#### Create a Node - `Libp2p.createLibp2p(options, callback)`
 
-> Creates an instance of the libp2p.Node.
+> Behaves exactly like `new Libp2p(options)`, but doesn't require a PeerInfo. One will be generated instead
+
+```js
+const { createLibp2p } = require('libp2p')
+createLibp2p(options, (err, libp2p) => {
+  libp2p.start(() => {})
```

Comment:
It may be worth adding a line to check the error here in case people blindly copy and paste this code

Reply:
Yeah I agree, the start callback should also probably due that. I'm thinking just a simple throw if the `err` exists to try and be overtly clear if there's an issue.

Reply:
Sounds good 👍